server/service/integration_mdm_test.go

```diff
@@ -6709,22 +6709,13 @@ func (s *integrationMDMTestSuite) TestValidSyncMLRequestNoAuth() {
 
 	resp := s.DoRaw("POST", targetEndpointURL, requestBytes, http.StatusOK)
 
-	resBytes, err := io.ReadAll(resp.Body)
+	_, err = io.ReadAll(resp.Body)
```

Comment:
Not sure if you meant to return to this test before merging (I see the TODO below), but just in case, those 2 lines 6712 and 6713 are unnecessary as it stands.

Reply:
just removed these two lines. Will bring it back when GetMDMWindowsManagementResponse() (management endpoint is in place)